pyproject.toml

```diff
@@ -38,6 +38,8 @@ pycdlib = "^1.12.0"
 randmac = "^0.1"
 toml = "^0.10.2"
 Pillow = {version="^9.0.0", optional=true}
+boto3 = "1.21.37"
```

Comment:
Mark aws related package with optional and create an "aws" extra for it

Reply:
Updated, thanks